src/bare-metal/aps/mmio.md

```diff
@@ -0,0 +1,17 @@
+# Volatile memory access for MMIO
+
+ * Use `pointer::read_volatile` and `pointer::write_volatile`.
```

Comment:
It seems this slide comes a bit too late? There was already talk about MMIO earlier on the "Raw MMIO" slide.

Reply:
The page before was more in the context of building up levels of abstraction for accessing microcontroller peripherals. This one is setting the scene for talking about driver abstractions for more general-purpose systems.